### ISSUE TITLE
Multiline echo with chaining

### ISSUE THREAD

**Issue by zepfietje** [Stillat/blade-parser-typescript#20]
Often long chains inside `{{ }}` are put on multiple lines.
However, after formatting it's all put on a single line.

Input
```blade
<label
    {{
        $attributes
            ->whereDoesntStartWith('wire:model')
            ->class(['flex gap-3 text-sm'])
    }}
>
</label>
```

Output
```blade
<label
    {{ $attributes->whereDoesntStartWith('wire:model')->class(['flex gap-3 text-sm']) }}
></label>
```

Expected input to remain unchanged or be something like:
```blade
<label {{
    $attributes
        ->whereDoesntStartWith('wire:model')
        ->class(['flex gap-3 text-sm'])
}}>
</label>
```

**Comment by JohnathonKoster** [comment]
The expected behavior of these things seems opposite to that of https://github.com/Stillat/blade-parser-typescript/issues/13

I'll see what I can do with the internal prettier stuff, though 🤔

**Comment by zepfietje** [comment]
Yeah it's tricky.
Have a look at the Filament codebase if you like.
We have a lot of places where we really need to put things on multiple lines.

**Comment by JohnathonKoster** [comment]
💡 I suppose I could check if the `{{` and `}}` were intentionally put on separate lines and swap that behavior internally

**Comment by zepfietje** [comment]
Interesting idea! Let me know it you want me to test anything.

**Comment by JohnathonKoster** [comment]
I've got the behavior much much better with this new concept (this new behavior will also maintain the current behavior of #13 ). For this input:

```blade
<label
    {{
        $attributes
            ->whereDoesntStartWith('wire:model')
            ->class(['flex gap-3 text-sm'])
    }}
>
</label>
```

it will produce:

```blade
<label
    {{ 
        $attributes
            ->whereDoesntStartWith("wire:model")
            ->class(["flex gap-3 text-sm"])
     }}
></label>

```

and for the following:

```blade
<div {{ $attributes->class([
    'some classes',
    match ($someCondition) {
        true => 'more classes foo bar baz',
        default => 'even more classes foo bar baz',
    },
]) }} more attributes class="one two three" something="else">
</div>
```

it produces

```blade
<div
    {{ 
        $attributes->class([
            "some classes",
            match ($someCondition) {
                true => "more classes foo bar baz",
                default => "even more classes foo bar baz"
            },
        ])
     }}
    more
    attributes
    class="one two three"
    something="else"
></div>

```

It will place the `{{` and `}}` on its own line and indent the contents relative to that (keeping the start/end inline with those was becoming a nightmare).

If that behavior looks good to you I'll tag a new version 🙂

**Comment by zepfietje** [comment]
Wow that looks perfect, John!
Tag that version, then I'll test it on my codebases.

We should be pretty close to a perfect formatter now!! 😄

**Comment by JohnathonKoster** [comment]
This is now available in release `1.1.4`
